src/store/store-provider.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
Is there any reason why you use `.jsx` file extension for all of your React components except this one?

Just wondering if there is something in the dev workflow that you prefer by using the `.jsx` extension, and maybe why it's not used here.

Reply:
That’s right, I was thinking about it in a normal store way, so I wouldn’t need to use HTML code inside this file, hence no need for jsx extension. But actually my implementation does include a Context provider wrapping other components, so I should have changed the extension to jsx.

Reply:
Resolved: fixed, https://github.com/rankun203/test-fullstack/commit/f46979eafe8bda9db2c36b0da6b7a221853cc086

Reply:
Fixed in https://github.com/rankun203/test-fullstack/commit/f46979eafe8bda9db2c36b0da6b7a221853cc086

Reply:
nice :)